asv_bench/benchmarks/reduce.py

```diff
@@ -1,44 +1,33 @@
 import numpy as np
-import numpy_groupies as npg
```

Comment:
I think this is wrong and you've copied an old version of the file.

Reply:
Hmm, might've had an old fork...